en/docs/learn/integration-tutorials/edi-transformation-with-ballerina.md

```
@@ -0,0 +1,359 @@
+        → Respond
+    ```
+
+    <a href="{{base_path}}/assets/img/develop/ballerina-module/finalView.png" class="glightbox"><img src="{{base_path}}/assets/img/develop/ballerina-module/finalView.png" alt="Final View" width="80%"></a>
+
+### Step 5: Run and test the integration
+
+1. Click the **Run** icon in VS Code to start the integration project.
+
+2. Open the integrated **Try it** feature or use a tool like Postman to send a `POST` request to `http://localhost:8290/transform` with the following EDI payload as plain text in the request body:
+
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)